openid-federation-1_0.xml

```diff
@@ -4458,7 +4458,7 @@ Host: openid.sunet.se
 
         <section anchor="tm-status-request" title="Trust Mark Status Request">
           <t>
-            The request MUST be an HTTP request using the POST method
+            The request MUST be an HTTP request using the GET method
```

Comment:
I ask to support both GET and POST to reduce breaking changes in the current implementations.

technically, from an implementation perspective, this is not a big change.

Reply:
@zachmann can you please apply @peppelinux's suggestion about supporting both GET and POST?

Reply:
If Trust Marks a re no longer being submitted, and we're left only with `sub` and `trust_mark_id`, I don't see reason to support POST in the long term.

Reply:
ok